### ISSUE TITLE
NIC type with "undionly" could NOT get "undionly.kpxe" but still "ipxe.kpxe"

### ISSUE THREAD

**Issue by lvii** [xbgmsharp/ipxe-buildweb#49]
Hi,

I have chozen the **NIC type** with **undionly**, but I still get a **large** `ipxe.kpxe` file.

Standard mode could generate a smaller `undionly.kpxe` file :

```
-rw-r--r--@ 1 me     wheel   331K  8  9 15:00 ipxe.kpxe  <-- size larger than undionly.kpxe
-rw-r--r--@ 1 me     wheel    65K  8  9 15:03 undionly.kpxe
```

![img](http://i.imgur.com/2OWMEGg.png)

**Comment by Sebastian-Roth** [comment]
Yeah, we ran into the same thing. So it still seems to be an issue. Is anyone working on this? Do we need to provide more information? Or is it just that we misunderstand the options giving in the interface?

**Comment by NiKiZe** [comment]
I would say that being able to build .kpxe for anything other then undionly would be a bug. Even tho it is possible to do, and _might_ work it is not supported in any way.
